lib/widgets/fields/padding_field/padding_field.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_quest/widgets/fields/padding_field/padding_types_layout.dart';
+
+class PaddingField extends StatefulWidget {
+  final void Function(EdgeInsets) onChanged;
+
+  const PaddingField({
+    Key? key, // Add the 'key' parameter
+    required this.onChanged,
+  }) : super(key: key);
+
+  @override
+  State<PaddingField> createState() => _PaddingFieldState();
+}
+
+class _PaddingFieldState extends State<PaddingField> {
+  PaddingType selectedOption = PaddingType.all;
+
+  @override
+  Widget build(BuildContext context) {
+    Widget paddingBoxType;
+
+    switch (selectedOption) {
+      case PaddingType.all:
+        paddingBoxType = PaddingTypeLayouts(
+          paddingType: PaddingType.all,
+          onChanged: widget.onChanged,
+        );
+      case PaddingType.only:
+        paddingBoxType = PaddingTypeLayouts(
+          paddingType: PaddingType.only,
+          onChanged: widget.onChanged,
+        );
+      case PaddingType.symmetric:
+        paddingBoxType = PaddingTypeLayouts(
+          paddingType: PaddingType.symmetric,
+          onChanged: widget.onChanged,
+        );
+    }
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            PaddingTypeSelectionRadioButton(
+              text: "All",
```

Comment:
This can be obtained using the extension as discussed in the article. Instead of providing text explicitly, use an extension getter title that provides title for a certain PaddingType. Also, the field name should be title and not text.